paper.md

```diff
@@ -0,0 +1,88 @@
+
+![Figure 1. A CT image correctly aligned to the MR image. It is worth noting that the darker areas in the MR are skull whereas the ligher, outermost regions are subcutaneous fat.](figures/Figure_1.png){ width=100% }
+
+Next, electrode contact locations are selected in merged CT-MR coordinates using a graphical user interface (GUI) implemented in PyQt. In the GUI, users click or scroll though slices to find contacts and then associate each contact location with a channel name in a menu. When a channel is associated with a location, it is rendered in the 3D view as well as marked on the slice plots.
+
+![Figure 2. A screenshot of the GUI during a sEEG electrode location. For sEEG, the 3D brain view is typically similar to the surgical plans, making it easier to determine which electrode is which than trying to piece together locations and directions from the three 2D slice images.](figures/Figure_2.png){ width=100% }
+
+![Figure 3. A screenshot of the GUI during an ECoG electrode location. For ECoG, the 3D brain view makes it easier to select each contact in a large grid. Without the 3D view, it is difficult to continue in one row without jumping to another, wrong row when the contacts are out of the plane in two directions. Mixing up rows makes it slow and difficult to correctly mark the contacts.](figures/Figure_3.png){ width=100% }
+
+Located electrode contacts in merged CT-MR coordinates for an individual subject can be morphed to a template brain for group analysis. This is done using a symmetric diffeomorphic registration [SDR; @GaryfallidisEtAl2014:2014]. This non-linear mapping tends to be more accurate than a linear transformation such as the Talairach transform [@DaleEtAl1999:1999] as shown in Figure 4.
+
+![Figure 4. a) Example sEEG electrodes in a 3D renderering of individual subject anatomy. b) The electrodes in ``a`` mapped to the Freesurfer ``fsaverage`` brain using the SDR morph. c) The electrodes in ``a`` mapped to to the Freesurfer ``fsaverage`` brain using the Talairach transform. Note that this linear transformation is less accurate than the SDR transform in ``b``.](figures/Figure_4.png){ width=100% }
+
+For ECoG electrodes, "brain shift", caused by changes in pressure during the craniotomy, can also be accounted for using MNE-Python. This shift causes grid and strip electrodes to be deeper in the post-operative CT than they were in the pre-operative MR. This makes it so that the electrodes are inside the pial surface in the pre-operative MRI. In MNE-Python, this is compensated for by projecting the grid to the leptomenigial surface.
+
+![Figure 5. a) A 3D rendering of an ECoG grid without correction for "brain shift". b) The same 3D rendering with the "brain shift" correction](figures/Figure_5.png){ width=100% }
```

Comment:
This image is a bit unclear. Is there a way to show it more clearly? Maybe a different angle? It is hard to tell it is beneath pial surface on B

Reply:
I don't think so, I tried several angles, it's a lot easier to see when you can manipulate it but if you only have one view this is about as good as you can do

Reply:
I could make the electrode sphere's completely opaque though, I think that might help

Reply:
I think that really helped, I made a new figure changing the opacity parameters